lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -148,6 +154,11 @@ $(document).ready(function() {
     }
   }
 
+  if (match = new RegExp(/\?redirected\_from=([^?]*)/).exec(window.location.href)) {
```

Comment:
This code could be nested under an if statement that checks whether we're viewing a page (one that tests for the existence of `div.markdown-body` or `div.wiki-content`? I forget which)

Reply:
`div.markdown-body`. Will move the code there.